js/src/lib/api/connectToCloud.ts

```diff
@@ -0,0 +1,11 @@
+import { axiosClient } from "./axiosClient";
+import { AxiosResponse } from "axios";
+
+export interface ConnectToCloud {
+  connection_url: string;
+}
+
+export async function connectToCloud(): Promise<ConnectToCloud> {
+  const data = await axiosClient.post<never, AxiosResponse<ConnectToCloud>>("/api/connect");
```

Comment:
you can rewrite this as `const data = await axiosClient.post<ConnectToCloud>("/api/connect");` -> Axios passes through the response. you can also remove the import line above